### PR TITLE
August referral contest

### PR DESCRIPTION
**Pull request by alesevero**
[August Referral Contest SaaSquatch/Product Change](https://musoraproduct.myjetbrains.com/youtrack/issue/CMT-77/August-Referral-Contest-SaaSquatch-Product-Change)

- Adds logic for the August Referral Contest

Branch is being used in MWP as those changes will be reverted after the contest ends

### CODE REVIEW

File: src/Services/OrderFormService.php [hunk 1]
```diff
@@ -16,7 +17,10 @@
 use Railroad\Ecommerce\Gateways\PayPalPaymentGateway;
 use Railroad\Ecommerce\Repositories\PaymentMethodRepository;
 use Railroad\Ecommerce\Repositories\PaymentRepository;
+use Railroad\Ecommerce\Repositories\ProductRepository;
 use Railroad\Ecommerce\Requests\OrderFormSubmitRequest;
+use Railroad\Referral\Models\Referrer;
```
Comment (BrandonKerr) [inline src/Services/OrderFormService.php:22]: Do we not need to include railroad/referral as a dependency in composer?

**Review by BrandonKerr — COMMENTED** [review]
Just one small question/request. Everything else looks good to me 👍

File: src/Services/OrderFormService.php (hunk 1 above)
Comment (alesevero) [reply src/Services/OrderFormService.php:22]: It was working without it, but I've added and did some testing, no conflicts between other dependencies. So just to make sure everything is correct, I'll keep the dependency in. Should be good now!

Comment (BrandonKerr) [reply src/Services/OrderFormService.php:22]: Thanks!

**Review by BrandonKerr — APPROVED** [review]
LGTM 👍

**Pull request merged**